rooms:
  - LIVINGROOM
  - KITCHEN
  - BEDROOM
  - TOILET

LIVINGROOM:
  entry_goal: {x: 1.46, y: 0.34, yaw: 0.0}
  cleaning_goals:
    - {x: 2.67, y: 0.23, yaw: 0.0}
    - {x: 6.72, y: 0.32, yaw: 0.0}
    - {x: 7.05, y: 4.08, yaw: 0.0}
    - {x: 2.75, y: 4.48, yaw: 0.0}
  qr_expected: "ROOM=LIVINGROOM"

KITCHEN:
  entry_goal: {x: 0.76, y: 0.409, yaw: 3.14}
  cleaning_goals:
    - {x: -4.59, y: 0.41, yaw: 0.0}
    - {x: -4.66, y: 4.37, yaw: 0.0}
    - {x: -0.65, y: 4.615, yaw: 0.0}
    - {x: -0.42, y: 0.48, yaw: 0.0}
  qr_expected: "ROOM=KITCHEN"

BEDROOM:
  entry_goal: {x: 6.06, y: 0.44, yaw: -1.57}
  cleaning_goals:
    - {x: 6.98, y: -0.97, yaw: 0.0}
    - {x: 6.66, y: -4.96, yaw: 0.0}
    - {x: 5.20, y: -4.74, yaw: 0.0}
    - {x: 5.21, y: -0.95, yaw: 0.0}
  qr_expected: "ROOM=BEDROOM"

TOILET:
  entry_goal: {x: 3.347, y: 4.589, yaw: 0.0}
  cleaning_goals:
    - {x: 0.39, y: 4.66, yaw: 0.0}
    - {x: 0.26, y: 1.27, yaw: 0.0}
    - {x: 1.88, y: 1.18, yaw: 0.0}
    - {x: 2.01, y: 4.48, yaw: 0.0}
  qr_expected: "ROOM=TOILET"

